webapp/crashstats/crashstats/tests/test_jinja_helpers.py

```diff
@@ -329,343 +313,448 @@ def test_with_unicode_signature(self):
-        assert quote_plus("0  test_module  foo::bar  foo.cpp\n") in url
+        qs = self._extract_query_string(url)
+        comment = (
+            "Crash report: http://localhost:8000/report/index/70dda764-a402-4ca3-b806-c38dd0240328\n"
+            "\n"
+            "Top 1 frames of crashing thread:\n"
+            "```\n"
+            "0  test_module  foo::bar(char* x, int y)  foo.cpp\n"
+            "```"
+        )
+        assert qs["comment"][0] == comment
 
     def test_comment_missing_file(self):
         """If a frame is missing file info, do not include it."""
-        req = RequestFactory().get("/report/index")
-        raw_crash = {}
-        report = self._create_report()
-        parsed_dump = self._create_dump(
-            threads=[
-                self._create_thread(
-                    frames=[
-                        self._create_frame(
-                            frame=0,
-                            module="test_module",
-                            signature="foo::bar(char* x, int y)",
-                            file=None,
-                            line=None,
-                        )
-                    ]
-                )
-            ]
+        report = self._create_report(
+            crashing_thread=0,
+            json_dump=self._create_dump(
+                threads=[
+                    self._create_thread(
+                        frames=[
+                            self._create_frame(
+                                frame=0,
+                                module="test_module",
+                                signature="foo::bar(char* x, int y)",
+                                file=None,
+                                line=None,
+                            )
+                        ]
+                    )
+                ]
+            ),
         )
         url = generate_create_bug_url(
-            req, self.TEMPLATE, raw_crash, report, parsed_dump, 0
+            f"http://localhost:8000/report/index/{self.CRASH_ID}",
+            self.TEMPLATE,
+            report,
+        )
+        qs = self._extract_query_string(url)
+        comment = (
+            "Crash report: http://localhost:8000/report/index/70dda764-a402-4ca3-b806-c38dd0240328\n"
+            "\n"
+            "Top 1 frames of crashing thread:\n"
+            "```\n"
+            "0  test_module  foo::bar(char* x, int y)\n"
+            "```"
         )
-        assert quote_plus("0  test_module  foo::bar  \n") in url
+        assert qs["comment"][0] == comment
 
     def test_comment_missing_everything(self):
         """If a frame is missing everything, do not throw an error."""
-        req = RequestFactory().get("/report/index")
-        raw_crash = {}
-        report = self._create_report()
-        parsed_dump = self._create_dump(threads=[self._create_thread(frames=[{}])])
-        generate_create_bug_url(req, self.TEMPLATE, raw_crash, report, parsed_dump, 0)
-
-    def test_comment_no_html_escaping(self):
```

Comment:
We needed this test to make sure that the rendered string from Jinja didn't escape all the HTML which is what Jinja does because Jinja expects to be rendering a string _for HTML_ and not _to be put into a url to prefill a bug comment form_. Given we're not using Jinja anymore, we don't need this.